### PR TITLE
Add switch ssh|console and show more details

### PR DESCRIPTION
**Pull request by majst01**

### CODE REVIEW

File: cmd/tableprinters/machine.go [hunk 1]
```diff
@@ -18,7 +18,7 @@ func (t *TablePrinter) MachineTable(data []*models.V1MachineResponse, wide bool)
 		rows [][]string
 	)
 
-	header := []string{"ID", "", "Last Event", "When", "Age", "Hostname", "Project", "Size", "Image", "Partition"}
+	header := []string{"ID", "", "Last Event", "When", "Age", "Hostname", "Project", "Size", "Image", "Partition", "Rack"}
```
Comment (Gerrit91) [inline cmd/tableprinters/machine.go:21]: IMHO wide should always contain all the fields from the normal output + additional fields.

File: cmd/tableprinters/switch.go [hunk 2]
```diff
@@ -59,8 +59,13 @@ func (t *TablePrinter) SwitchTable(data []*models.V1SwitchResponse, wide bool) (
 			mode = "operational"
 		}
 
+		os := ""
+		if s.Os != nil {
+			os = s.Os.Vendor + "/" + s.Os.Version
```
Comment (Gerrit91) [inline cmd/tableprinters/switch.go:64]: Can look iffy when things like version could not be figured out?

File: cmd/tableprinters/machine.go (hunk 1 above)
Comment (majst01) [reply cmd/tableprinters/machine.go:21]: will do

File: cmd/tableprinters/switch.go (hunk 2 above)
Comment (majst01) [reply cmd/tableprinters/switch.go]: done

File: cmd/tableprinters/machine.go (hunk 1 above)
Comment (majst01) [reply cmd/tableprinters/machine.go:21]: Done

Comment (Gerrit91) [reply cmd/tableprinters/machine.go:21]: For the IPMI printer I can see it, but here it is still missing?

Comment (majst01) [reply cmd/tableprinters/machine.go:21]: stupid me

Comment (majst01) [reply cmd/tableprinters/machine.go:21]: done

**Comment by Gerrit91** [comment]
Depends on https://github.com/metal-stack/metal-api/pull/351.

**Review by Gerrit91 — APPROVED** [review]

**Pull request merged**